src/constants/style-related.ts

```diff
@@ -1,4 +1,4 @@
-export const VALIDATION_CONTEXTS = ['success', 'warning', 'danger'] as const;
+export const VALIDATION_CONTEXTS = ['success', 'warning', 'danger', 'primary'] as const;
```

Comment:
why is this change?

Reply:
We use Context type for context and this test case contains  primary for actions

Reply:
Should we add it or just change actions data ?

Reply:
I'm not sure what test you referring to. This is a global constant that will effect all components using context, isn't it?

Reply:
@eszthoff yes these test actions 
![image](https://user-images.githubusercontent.com/104063210/235183670-214f356e-b031-4e4f-bf44-9a5818940eec.png)

Reply:
I can change it to warning or should we use here primary ?

Reply:
I think this context has been removed with the color changes, and the test wasn't fixed. I like your suggestion to change the test to `warning`

Reply:
:+1: